site/assets/scss/_sidebar.scss

```diff
@@ -11,6 +11,23 @@
     margin-left: -.25rem;
     overflow-y: auto;
   }
+
+  @include media-breakpoint-down(lg) {
+    .offcanvas-lg {
+      border-right-color: var(--bs-border-color);
```

Comment:
Feels weird to use this since the color used is black (compared to `$gray-500` in side navigation, offcanvases, ...)

Reply:
"Boosted moded" it via https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1757/commits/a543691d0de90f7c0a1ec520fdd95819f21c7d12